frontend/src/pages/__snapshots__/RunDetails.test.tsx.snap

```diff
@@ -1138,6 +1138,12 @@ exports[`RunDetails renders an empty run 1`] = `
 </div>
 `;
 
+exports[`RunDetails shows a one-node compressed workflow graph 1`] = `
+<div
+  className="page"
+/>
```

Comment:
sth seems wrong, the snapshot is empty.
If you compare it to snapshot of the one node test, it has some content